### DISCUSSION TITLE
Encounter error when trying to build the docket image for netbox 3.5.9

### DISCUSSION THREAD

**Discussion by richard063** [netbox-community/netbox-docker#1060] [Getting Help]
Hello  all, 

May i seek your help and expertise on this topic as i'm encounter the error as below when i try to build the netbox 3.5.9 image for me to upload to my AWS ECR.

i can fire up the netbox and run locally, but when i try to build the image locally before upload to ecr, i encounter this error..
user11@user11-virtual-machine:~/netbox-docker$ sudo docker build -t netboc359 .
[sudo] password for user11: 
Sending build context to Docker daemon  97.79kB
Step 1/26 : ARG FROM
Step 2/26 : FROM ${FROM} as builder
base name (${FROM}) should not be blank
user11@user11-virtual-machine:~/netbox-docker$ 

i try to google and search for this topic, i dont seems to able find anything, so, i'm pretty sure it's my problem.. but i dont know where and how to solve it, hence asking here.

**Comment by kkthxbye-code** [comment]
Use the build script:

https://github.com/netbox-community/netbox-docker/wiki/Build#build-the-image-from-scratch

**Reply by richard063** [reply]
thanks for your sharing, but i can't get my head around with the build.sh script...

**Reply by kkthxbye-code** [reply]
Then it's very likely that you do not actually need to build the image from scratch. Could you provide the reason you want to build the image yourself?

**Reply by richard063** [reply]
Hello , i think youa re absolutely right...  i clone the image and its working. thank you.
